tetrateci/create_istio_release.sh

```diff
@@ -101,18 +101,13 @@ if [[ -z $TEST ]]; then
 
     go run main.go build --manifest manifest.archive.yaml
 
+    python3 -m pip install --upgrade cloudsmith-cli --user
```

Comment:
can we use
```
pip install --upgrade cloudsmith-cli
cloudsmith push raw ${CLOUDSMITH_USER}/getistio $package
```

Reply:
unsure if `pip3`, if not, you might also need to
```
curl https://bootstrap.pypa.io/pip/2.7/get-pip.py --output get-pip.py \
  && python2 get-pip.py \
  && pip install --upgrade cloudsmith-cli
```

Reply:
I am not sure that the cloudsmith-cli runs on python2 or not, just to be extra cautious I added `python3 -m`  since the runner has both python2 and python3

Reply:
sure as long as it works on the CI instance, sgtm :)

Reply:
Ahh I get what you are trying to say, my bad, I have updated all the pip calls now, 😓